receipt_agent/receipt_agent/agents/place_id_finder/graph.py

```
@@ -438,20 +440,28 @@ async def run_place_id_finder(
         }
 
-        # Add LangSmith metadata if tracing is enabled
+        # Add LangSmith metadata and callbacks if tracing is enabled
         if os.environ.get("LANGCHAIN_TRACING_V2") == "true":
             config["metadata"] = {
                 "image_id": image_id,
                 "receipt_id": receipt_id,
                 "workflow": "place_id_finder",
             }
+            # Use project name from environment (same as parent trace)
+            project_name = os.environ.get("LANGCHAIN_PROJECT", "receipt-label-validation")
```

Ruff (0.14.13)

Local variable `project_name` is assigned to but never used

Remove assignment to unused variable `project_name`

(F841)